go.mod

```diff
@@ -4,99 +4,102 @@ go 1.17
 
```

Comment:
IMO, we need to revisit how we updated these modules. I suggest to re-evaluate each modules here and check if we really need to bump them or not. I can see a lot of non-`//indirect` modules here which means estuary `directly` uses them which can cause regression issues.